ldappool/__init__.py

```
@@ -0,0 +1,450 @@
+                        return True
+                except Exception as ldaperr:  # collect with parent exception
+                    pass
+                logging.error(
+                    "LDAP exception pinging server %s %s", self.uri.initializeUrl(), ldaperr  # type: ignore
+                )
+                raise ldaperr  # type: ignore
+        return True
+
+    def get(self, binddn: str = "", bindpw: str = ""):
+        if len(self._pool) == 0:
+            self.scale
```

Ruff (0.14.8)

Found useless expression. Either assign it to a variable or remove it.

(B018)